tests/Resources/ODS.robot

```diff
@@ -149,3 +149,14 @@ Get RHODS URL From OpenShift Using UI
     ...     //a[@data-test="application-launcher-item" and starts-with(@href,'https://rhods')]
     ${href}  Get Element Attribute    ${link_elements}    href
     [Return]   ${href}
+
+Disable Access To Grafana Using OpenShift Port Forwarding
+    [Documentation]   Kill process running in background based on Id
+    [Arguments]  ${PROC}
+    Close Browser
```

Comment:
if we want to keep this kw as common kw, maybe `Close Browser` should be moved out. If you need it for the teardown, you could create a new teardown kw or you could use the suite teardown to close the browser

Reply:
ack done changes